practitioner-guides/diverse-leadership.md

```diff
@@ -0,0 +1,130 @@
+
+
+# Step 2: Diagnosis
+
+When considering diverse leadership, it can be difficult to track and gather specific data due to privacy concerns and the difficulty in storing personally identifiable data. Rather than trying to gather data directly from the people in your community to determine leadership diversity, this might be better diagnosed via a community perception survey to ask the broader community how they feel about the diversity of leadership within the project. 
+Here are a few sample questions that you might use in a community perception survey:
+
+Board/council diversity:
+
+* Likert scale [1-x] item: I feel represented by the board or council.
+* Likert scale [1-x] item: The board or council represents the diversity of the project community.
+* Likert scale [1-x] item: The board or council considers diverse backgrounds and experiences.
+
+Sponsorship:
+
+* Likert scale [1-x] item: I am sponsoring other members.
+* Likert scale [1-x] item: I have a sponsor within the community who puts their reputation on the line to advocate for me.
+* Likert scale [1-x] item: The outcome of my sponsorship is immensely rewarding. 
+
+Inclusive Leadership:
+
+* Likert scale [1-x] item: I understand the process for moving into a leadership role.
+* Likert scale [1-x] item: The project values and recognizes different types of experts equally (e.g., technical and non-technical)
+* Likert scale [1-x] item: Do the project’s leaders agree to a standard by which they can be held accountable?
+
+The web pages linked above for each of these metrics contain additional survey questions and will likely give you ideas for other questions that would be appropriate to better understand these topics within your community.
+You should also ask some open-ended questions where people can provide suggestions for how to improve your leadership diversity. This practitioner guide is intended to get you started; feel free to implement the practices described in your context.
+
+
+# Step 3: Gather Additional Data if Needed
+
+CHAOSS has other metrics related to diversity that can help diagnose specific problems within your community.
+
+Additional Metrics: 
+
+* [Mentorship](https://chaoss.community/?p=3524)
+* [Code of Conduct for a Project](https://chaoss.community/?p=3517)
+* [Psychological Safety](https://chaoss.community/?p=3544)
+* [Project Burnout](https://chaoss.community/?p=3537)
+
+
+# Step 4: Make Improvements
```

Comment:
The text in this section is squished together - I think you need to add a blank line between each paragraph.